### PR TITLE
4.view a list of all reviews

### PR DESCRIPTION
**Pull request by gareth5james**

### CODE REVIEW

**Comment by gareth5james** [comment]
https://excalidraw.com/#json=3Ra1Ii-nFZvOqPdneKVju,ra-HZIkBiJdn7x6uCJjqeA

File: nc-games/src/App.css [hunk 1]
```diff
@@ -14,25 +15,51 @@
...
+
+.reviewList-Header {
+  border: 2px solid #73648E;
+  margin: 5px;
+}
+
+.reviewList-Body {
+  border: 2px solid #73648E;
+  margin: 5px;
+  overflow:scroll;
```
Comment (popatre) [inline nc-games/src/App.css]: Its a small thing - but css naming you'll more commonly see with a double underscore - BEM - https://www.devbridge.com/articles/implementing-clean-css-bem-method/#:~:text=What%20is%20BEM%3F,by%20following%20some%20simple%20rules.

File: nc-games/src/Components/Reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { useState, useEffect } from "react"
+
+function Reviews() {
+    const [items, setItems] = useState([])
+
+    useEffect(() => {
+        getReviews().then(data => {
+            setItems(data)
+        })
+    }, [])
```
Comment (popatre) [inline nc-games/src/Components/Reviews.jsx]: Needs loading to be handled please.

**Review by popatre — CHANGES_REQUESTED** [review]
Could you handle the loading for your APi request please - some feedback for the user

**Comment by gareth5james** [comment]
updated

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Looks fantastic! 👍

**Pull request merged**